models/keyword.go

```diff
@@ -59,20 +93,25 @@ func CountAllKeyword(query map[string]interface{}) (totalRows int64, err error)
 	return querySetter.Count()
 }
 
-// GetKeyword retrieves a Keyword by query list then returns error if it doesn't exist
-func GetKeyword(query map[string]interface{}) (keyword *Keyword, err error) {
+// GetStatusKeyword returns an integer mapping with readable key
+func GetStatusKeyword(status string) int {
```

Comment:
Related to the comment above, placing status after keyword would be more logical:

```suggestion
func GetKeywordStatus(status string) int {
```